get-started/deployment-options.md

```diff
@@ -1,17 +1,21 @@
 ---
+navigation_title: Deployment options
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/elasticsearch-intro-deploy.html
 products:
   - id: elasticsearch
 ---
 
-# Deployment options [elasticsearch-intro-deploy]
+# Deployment options
 
-Use this page for a quick overview of your options for deploying Elastic.
+You can run Elastic on virtually any infrastructure, allowing you to choose the model that best fits your operational needs. 
 
-:::{include} /deploy-manage/_snippets/deployment-options-overview.md
-:::
+These are the most common deployment types and their main features:
 
-:::{tip}
-Refer to [](/deploy-manage/index.md) for detailed information and guidance on choosing the right deployment for your needs.
-:::
+* **{{serverless-full}}**: This fully managed SaaS offering abstracts away all underlying infrastructure, automatically and seamlessly scaling resources to meet your workload demands. It's designed for operational simplicity, with usage-based pricing that allows you to focus on your data without managing clusters.  
```

Comment:
would like to see all of these link into their core docs.

I would also like to see serverless + ech explicitly mention [elastic cloud](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2795/deploy-manage/deploy/elastic-cloud) (i.e. "SaaS offering managed through [Elastic Cloud]"), because it's its own layer that people need to sign up for and manage, and having a cloud org lets you have both serverless and ech deployments

Reply:
serverless also allows you your choice of cloud provider from AWS, GCP, and Azure.